rest_framework_jwt/compat.py

```diff
@@ -1,11 +1,35 @@
 import rest_framework
+
+from django.db import models
+
 from distutils.version import StrictVersion
+from functools import partial
+
+from rest_framework import serializers
 
 
 if StrictVersion(rest_framework.VERSION) < StrictVersion('3.0.0'):
     from rest_framework.serializers import Serializer
 else:
-    class Serializer(rest_framework.serializers.Serializer):
+    class Serializer(serializers.Serializer):
         @property
         def object(self):
             return self.validated_data
+
+
+def get_uuid_field():
```

Comment:
I'm not really excited about introducing an optional requirement here just for a UUIDField. I think we can just simplify this for built-in Django 1.8 UUIDField and CharField for anything less.